tweet_recommendations/embeddings/sif_embedding.py

```diff
@@ -14,19 +14,33 @@
 
 class SIFEmbedding(Method):
 
-    def __init__(self, path_to_keyedvectors_model: str, verbose: bool = False):
+    def __init__(self, path_to_keyedvectors_model: str,
+                 embedding_name: Optional[str] = "embedding",
+                 verbose: bool = False):
         """
         Method for creating embeddings out of words and combining them into single tweet embedding with modified
         weighted average of those embeddings.
         :param path_to_keyedvectors_model: Path to converted by script `convert_embedding_model_to_mmap.py` gensim
             model. It can either word2vec or fasttext, `gensim` handles both.
+        :param embedding_name: Optional, name of used embedding method.
+                Required when `path_to_keyedvectors_model` is not provided.
```

Comment:
"Required when `path_to_keyedvectors_model` is not provided.", ale argument `path_to_keyedvectors_model` **nie** jest opcjonalny w sygnaturze `__init__`'a